src/main/java/spotify/api/impl/BrowseApiRetrofit.java

```diff
@@ -152,6 +214,34 @@ public AlbumSimplifiedPaging getNewReleases(Map<String, String> options) {
         }
     }
 
+    @Override
+    public Paging<AlbumSimplified> getNewReleasesUnwrapped(Map<String, String> options) {
```

Comment:
Same as the comments at `getCategoryPlaylistsUnwrapped`

Reply:
The requested changes have been implemented.